package.json

```diff
@@ -33,28 +33,28 @@
     "node": ">=12"
   },
   "dependencies": {
-    "@google-cloud/datastore": "6.6.0",
-    "@google-cloud/secret-manager": "3.10.1",
-    "@google-cloud/storage": "5.15.3",
-    "@verdaccio/commons-api": "10.0.1",
-    "@verdaccio/streams": "10.0.0"
+    "@google-cloud/datastore": "7.2.0",
+    "@google-cloud/secret-manager": "4.2.0",
+    "@google-cloud/storage": "6.9.2",
+    "@verdaccio/commons-api": "10.2.0",
+    "@verdaccio/streams": "10.2.0"
   },
   "devDependencies": {
-    "@types/jest": "26.0.24",
-    "@types/node": "14.14.31",
+    "@types/jest": "29.4.0",
+    "@types/node": "18.13.0",
     "@verdaccio/babel-preset": "10.0.0",
     "@verdaccio/eslint-config": "10.0.0",
-    "@verdaccio/types": "10.0.0",
+    "@verdaccio/types": "10.7.0",
     "babel-eslint": "10.1.0",
     "cross-env": "7.0.3",
     "eslint": "6.8.0",
-    "jest": "26.0.1",
-    "lint-staged": "9.5.0",
-    "prettier": "1.19.1",
-    "typescript": "3.9.10"
+    "jest": "29.4.2",
+    "lint-staged": "13.1.2",
+    "prettier": "2.8.4",
+    "typescript": "4.9.5"
   },
   "optionalDependencies": {
-    "fast-crc32c": "1.0.7"
+    "fast-crc32c": "2.0.0"
```

Comment:
This is no longer needed/used since `@google-cloud/storage` v6 (see googleapis/nodejs-storage#2003), so we should remove the dependency altogether.